examples/large_models/deepspeed_mii/LLM/model-config.yaml

```diff
@@ -0,0 +1,20 @@
+# TorchServe frontend parameters
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 100
+responseTimeout: 1200
+parallelType: "tp"
+deviceType: "gpu"
+# example of user specified GPU deviceIds
+deviceIds: [0,1,2,3] # seting CUDA_VISIBLE_DEVICES
```

Comment:
Is this working if we have 8 gpus in a host?

Reply:
yes, TorchServe support customer specified gpu allocation.